import-docker-worker-secrets.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash -e
```

Comment:
I'm still confused how this script works. What is `deploy/bin` by the way?

Reply:
It might be easier just to do this one time and commit to password store directly. You can have a note there that things need to be kept in sync manually?